CSS/CSS.sublime-syntax

```diff
@@ -110,6 +110,10 @@ contexts:
       # Hex Color
     - match: '(#)(\h{3}|\h{6})\b'
       scope: constant.other.color.rgb-value.css
+      # RGBA Hexadecimal Colors
```

Comment:
it looks like this change introduces a regression in that capture group 1 (`#`) is no longer being scoped as `punctuation.definition.constant.css` for 3 or 6 hex digit colors.